content/en/ddsql_reference/ddsql_default.md

```diff
@@ -460,6 +460,12 @@ SELECT *
 FROM k8s.daemonsets da INNER JOIN k8s.deployments de
 ON da.tags = de.tags -- for a specific tag: da.tags->'app' = de.tags->'app'
 ```
+### JSON functions and operators
+
+| Name | Return type | Description |
+|------|-------------|-------------|
+| json_extract_path_text(text json, text path…) | text | Extracts the JSON sub-object in JSON as text, defined by the path. Its behavior is equivalent to the [postgres function with the same name][2]. For example, `json_extract_path_text(col, ‘forest')` returns the value of the key `forest` for each JSON object in `col`. See the example below for a JSON array syntax.|
```

Comment:
```suggestion
| json_extract_path_text(text json, text path…) | text | Extracts a JSON sub-object as text, defined by the path. Its behavior is equivalent to the [Postgres function with the same name][2]. For example, `json_extract_path_text(col, ‘forest')` returns the value of the key `forest` for each JSON object in `col`. See the example below for a JSON array syntax.|
```

Reply:
Not sure if it's necessary to say the JSON sub-object is in JSON, but feel free to ignore that part of this suggestion if that's needed for some reason.

Additionally, the `[2]` reference link here might not be going to the expected location, as there's no reference to `json_extract_path_text` on that page. It seems like this might be the correct page:
https://www.postgresql.org/docs/current/functions-json.html